scripts/get-jwks-keys/main.go

```diff
@@ -24,7 +24,7 @@ func main() {
 		os.Exit(2)
 	}
 
-	jwksRSAKeys, err := jwksHandler.GetJWKSRSAKeys(REGION, USER_POOL_ID)
+	jwksRSAKeys, err := jwksHandler.GetJWKSRSAKeys(REGION, userPoolID)
```

Comment:
Fix region variable too
```suggestion
	jwksRSAKeys, err := jwksHandler.GetJWKSRSAKeys(region, userPoolID)
```